server/shared/types/dataset.ts

```diff
@@ -598,6 +598,7 @@ type Mds3Queries = {
 	singleSampleGenomeQuantification?: SingleSampleGenomeQuantification
 	singleSampleGbtk?: SingleSampleGbtk
 	DZImages?: any
+	WSImages?: any
```

Comment:
should declare obj as {imageBySampleFolder, type} and clarify purpose of each

Reply:
Fixed.